q2_longitudinal/_utilities.py

```diff
@@ -27,28 +27,40 @@
 TEMPLATES = pkg_resources.resource_filename('q2_longitudinal', 'assets')
 
 
+def _validate_input_values(state_1, state_2):
+    if state_1 == state_2:
+        raise ValueError((
+            'You have chosen the same value for state_1 and state_2. These '
+            'parameters must be given different values.'))
+
+
 def _get_group_pairs(df, group_value, individual_id_column='SubjectID',
                      group_column='Group', state_column='time_point',
-                     state_values=['1', '2'], drop_duplicates=True):
+                     state_values=['1', '2'], replicate_handling='error'):
     results = []
     group_members = df[group_column] == group_value
     group_md = df[group_members]
     for individual_id in set(group_md[individual_id_column]):
         result = []
         for state_value in state_values:
             state_value = df[state_column].dtype.type(state_value)
-            individual_id = \
-                df[individual_id_column].dtype.type(individual_id)
+            individual_id = df[individual_id_column].dtype.type(individual_id)
             _state = df[state_column] == state_value
             _ind = df[individual_id_column] == individual_id
             individual_at_state_idx = group_md[_state & _ind].index
             if len(individual_at_state_idx) > 1:
                 print("Multiple values for {0} {1} at {2} {3} ({4})".format(
```

Comment:
These `print` statements aren't going to be very useful here, since with some interfaces users will see them, and with others they won't. In general, `print` statements shouldn't be in any QIIME actions. Instead, these types of warnings should be included in the visualization - for example, see the yellow warning box in [this viz](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fdocs.qiime2.org%2F2017.7%2Fdata%2Ftutorials%2Fmoving-pictures%2Fcore-metrics-results%2Ffaith-pd-group-significance.qzv).

Since I know you don't have a lot of time to work on this before you leave, could you create an issue to replace these `print` statements with warnings in the visualization?